### ISSUE TITLE
Make `git push` work

### ISSUE THREAD

**Issue by cjb** [cjb/GitTorrent#3]
It should: 
* announce the new ref
* make a new mutable value and publish that

**Comment by phuicy** [comment]
Announce how? where? Forking?

I am interested in contributing towards this. But I feel that some coordination for work is necessary.

**Comment by cjb** [comment]
@phuicy "Announce" is a BitTorrent term for letting the network know that you are available to peer with for a specified hash.

**Comment by phuicy** [comment]
I realise that, but why do it for every push? Could you not do something simpler, auto track the bare repo? why not do it for a fork only?

**Comment by annanay25** [comment]
No, not a new torrent for every commit. But, we need to be able to tell the torrent tracker that a new node is available for peer. As @cjb said, for a specified hash.
